docs/installation/language.md

```diff
@@ -27,7 +27,7 @@ IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.
 In [1]: import helics
 
 In [2]: helics.helicsGetVersion()
-Out[2]: '1.0.2 (04-28-18)'
+Out[2]: '1.x.x (XX-XX-XX)'
```

Comment:
should probably have an x as the first number too since that will be updated soon